src/program.js

```diff
@@ -123,11 +125,21 @@ export class Program {
   }
 }
 
+declare var WEBEXT_BUILD_ENV: string;
 
-export function defaultVersionGetter(absolutePackageDir: string): string {
-  let packageData: any = readFileSync(
-    path.join(absolutePackageDir, 'package.json'));
-  return JSON.parse(packageData).version;
+export function defaultVersionGetter(
+  absolutePackageDir: string,
+  localEnv: string = WEBEXT_BUILD_ENV
```

Comment:
Since this is an optional argument, it should be defined as a keyword argument instead. This help makes the calls easier to read and it makes it easy to add more options later on. Example:
````javascript
export function defaultVersionGetter(
  absolutePackageDir: string,
  {localEnv: string = WEBEXT_BUILD_ENV} = {}
): string {
````